db/migrate/20180206164119_create_licenses.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLicenses < ActiveRecord::Migration[5.1]
+  def change
+    create_table :licenses, id: :uuid do |t|
```

Comment:
Like the UUID IDs... found this which could help save us in future? https://blog.bigbinary.com/2016/04/04/rails-5-provides-application-config-to-use-UUID-as-primary-key.html

Reply:
oh that's neat.  yep I'll use that

Reply:
Hm, so that doesn't quite do what we'd though.  Adding the :orm config just tell the migration helper to auto-add the ` id: :uuid` to a table definition when it makes the skeleton file.

Which is still worth doing but doesn't mean we can remove the clause from the existing files

Reply:
https://lab.io/articles/2017/04/13/uuids-rails-5-1/ explains it a bit more